### DISCUSSION TITLE
manage.py invalidate all from Netbox 3.0

### DISCUSSION THREAD

**Discussion by smutel** [netbox-community/netbox#8934] [Q&A] [answered]
Hello,

In Netbox older than version 3.0.0, the script manage.py has an option invalidate all.
From Netbox 3.0.0, this option disappeared.
Any other command to clear items after an update for example ?

Thanks.

**Comment by jeremystretch** [comment, marked as answer]
The management command was removed because automatic queryset caching was removed, due to all the bugs it was causing (see #6639). There is no replacement command because it is no longer needed.

**Reply by smutel** [reply]
Thanks for this information.

After upgrading from 2.11.12 to 3.0.12, I encountered issues during 1-2 minutes (error message about static, refresh loop, ...).
After a while it was good ...

Do you see something to add there (at the end of the file): https://github.com/smutel/ansible-role-netbox/blob/Netbox3.0/tasks/deploy_netbox.yml.

We already have: collectstatic / clearsessions / remove_stale_contenttypes.

**Reply by candlerb** [reply]
If you're talking about problems *during* the upgrade and until Netbox has finished restarting, this is to be expected.

The upgrade process modifies the database schema, meaning that the running instance of Netbox will no longer work properly.  Personally what I do is:
* stop netbox
* run the upgrade
* start netbox
rather than leave a broken instance running.

If that's not what you mean, then you'll need to be more specific about exactly what errors you see and under what circumstances.

**Reply by smutel** [reply]
I will try to reproduce this behavior and to explain it with more details. Thanks.
